Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json

```diff
@@ -567,6 +568,30 @@
           0.1
         ]
       ]
+    },
+    {
+        "name": "klubskub",
+        "delays": [
+            [
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.1,
+                0.5
```

Comment:
Could you make the final frame last a bit longer? Should be less distracting that way with the looping and people will be able to get a good read on the full text. Maybe 5-6 seconds. Could take a look through other signs for examples. Otherwise looks good.